src/main/resources/application.yml

```diff
@@ -5,17 +5,27 @@ logging:
     ROOT: INFO
     org.springframework.web: INFO
 nakadi:
+  topic:
+    default:
+      partitionNum: 8
+      replicaFactor: 1
```

Comment:
In local configuration it is 1 instead of 3 because in local environment we have only one kafka broker. This should be overridden with env var when deploying to AWS.